plugins/gtav/Game.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef GTAV_GAME
+#define GTAV_GAME
```

Comment:
```suggestion
#ifndef GTAV_GAME_H_
#define GTAV_GAME_H_
```
(Just a convention we follow for the Mumble source code)

Reply:
The `_H_` suffix was missing in amongus/Game.h too.